### ISSUE TITLE
Update to jQuery v3.4.0 (Bootstrap 4.x)

### ISSUE THREAD

**Issue by coliff** [twbs/bootstrap#28639]
Bootstrap v4 usually uses the latest available version of jQuery 3.x.

jQuery 3.4.0 is now available - can we update CDN links to use this as it has a minor vulnerability fix and a few performance improvements.

REF: https://blog.jquery.com/2019/04/10/jquery-3-4-0-released/

**Comment by herbalite** [comment]
Got a bit bigger in size though

**Comment by coliff** [comment]
right, but after gzip the difference is tiny - just 0.4 KB - and using v3.4.0 means you can get a higher score in Google Lighthouse tests thanks to no known vulnerabilities (https://snyk.io/vuln/npm:jquery) :-)

**Comment by pereorga** [comment]
Slim version is smaller: 
```
$ curl --compressed -so /dev/null http://code.jquery.com/jquery-3.4.0.slim.min.js -w '%{size_download}\n'
24328

$ curl --compressed -so /dev/null http://code.jquery.com/jquery-3.3.1.slim.min.js -w '%{size_download}\n'
28008
```

**Comment by XhmikosR** [comment]
Not feasible, there's an upstream bug, waiting for 3.4.1 in #28589

**Comment by XhmikosR** [comment]
For reference https://github.com/jquery/jquery/issues/4350

**Comment by XhmikosR** [comment]
Re-opening to track this, should be closed when #28589 is merged.

**Comment by XhmikosR** [comment]
Still waiting for jQuery v3.4.1, and now there's a security vulnerability also for < 3.4.0. But since this breaks our tests, we need to wait for 3.4.1.

**Comment by coliff** [comment]
Yay. https://blog.jquery.com/2019/05/01/jquery-3-4-1-triggering-focus-events-in-ie-and-finding-root-elements-in-ios-10/

**Comment by XhmikosR** [comment]
Thanks, I'm gonna take care of it for all branches later.

**Comment by XhmikosR** [comment]
Fixed by #28717
